src/main/java/com/example/housingmanagement/MyCommandLineRunner.java

```diff
@@ -0,0 +1,75 @@
+package com.example.housingmanagement;
+
+import com.example.housingmanagement.api.dbentities.Gender;
+import com.example.housingmanagement.api.dbentities.HouseInternalEntity;
+import com.example.housingmanagement.api.dbentities.OccupantInternalEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+@Component
+public class MyCommandLineRunner implements CommandLineRunner {
```

Comment:
To cudo docelowo zniknie - ale fajnie że poznałeś sposób, że można 'wbić się' ze swoją logiką w proces startowania aplikacji Springowej <3